programs/svm-spoke/src/instructions/token_bridge.rs

```diff
@@ -40,28 +40,28 @@ pub struct BridgeTokensToHubPool<'info> {
     pub token_messenger_minter_sender_authority: UncheckedAccount<'info>,
 
     /// CHECK: MessageTransmitter is checked in CCTP. Seeds must be \["message_transmitter"\] (CCTP Message Transmitter
-    /// program).
+    // program).
```

Comment:
Turning these into double slashes doesn't remove them from docs?

Reply:
it should remove them from the docs, aren't they?

Reply:
Okay, yes, before your change, Anchor was adding program). to the documentation.
Now, it doesn’t add documentation for this account (expected behavior).

Before your fix:
```
        {
          "name": "message_transmitter",
          "docs": [
            "program)."
          ],
          "writable": true
        },
```

After fix:
```
        {
          "name": "message_transmitter",
          "writable": true
        },
```

All good!